barchart.go

```diff
@@ -32,11 +32,21 @@ var (
 		Horizontal:             false,
 		VerticalBarCharacter:   "██",
 		HorizontalBarCharacter: "█",
-		Height:                 GetTerminalHeight() * 2 / 3,
-		Width:                  GetTerminalWidth() * 2 / 3,
+		Height:                 GetDefaultBarChartWidth(),
+		Width:                  GetDefaultBarChartHeight(),
 	}
 )
 
+// GetDefaultBarChartWidth calculates the width of DefaultBarChart.
+func GetDefaultBarChartWidth() int {
+	return GetTerminalHeight() * 2 / 3
+}
+
+// GetDefaultBarChartHeight calculates the height of DefaultBarChart.
+func GetDefaultBarChartHeight() int {
+	return GetTerminalWidth() * 2 / 3
+}
+
```

Comment:
I think we should remove those methods and inline them for now, until we have a solid terminal rezising approach :)